.github/workflows/mobile-compile_time_options.yml

```diff
@@ -103,23 +103,25 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
-        cd mobile && ./bazelw build \
-        $([ -z $GITHUB_TOKEN ] || echo "--config=remote-ci-macos") \
-          --fat_apk_cpu=x86_64 \
-          --define=admin_html=enabled \
-          --define=envoy_mobile_request_compression=disabled \
-          --define=envoy_enable_http_datagrams=disabled \
-          --define=google_grpc=disabled \
-          --define=envoy_yaml=disabled \
-          --@com_envoyproxy_protoc_gen_validate//bazel:template-flavor= \
-      //:android_dist && ./bazelw test \
-          --fat_apk_cpu=x86_64 \
-          --define=admin_html=enabled \
-          --define=envoy_mobile_request_compression=disabled \
-          --define=envoy_enable_http_datagrams=disabled \
-          --define=google_grpc=disabled \
-          --define=envoy_yaml=disabled \
-          --@com_envoyproxy_protoc_gen_validate//bazel:template-flavor= \
-      //test/java/integration:android_engine_socket_tag_test \
-      //test/java/integration:android_engine_start_test \
-      //test/java/io/envoyproxy/envoymobile/utilities:certificate_verification_tests
+        cd mobile \
+        && ./bazelw build \
+            $([ -z $GITHUB_TOKEN ] || echo "--config=remote-ci-macos") \
```

Comment:
should we copy this line down to the test section while we're in here?

also I'm having trouble seeing what changed other than whitespace

Reply:
me too - but i was hitting this problem when testing - ill test again